datalad/interface/common_cfg.py

```diff
@@ -197,6 +200,18 @@
                'title': 'Does not execute teardown_package which cleans up temp files and directories created by tests if this flag is set'}),
         'type': EnsureBool(),
     },
+    'datalad.tests.force-fake-home': {
+        'ui': ('yesno', {
+               'title': 'Force running tests on OSX and window with manipulated HOME or USERPROFILE respectively.',
```

Comment:
See comment above. This somewhat conflates what the setting is about, and how it is put to action.

```suggestion
               'title': 'Force running tests on OSX and window with manipulated HOME or USERPROFILE respectively',
```